content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md

```diff
@@ -0,0 +1,293 @@
+<br />
+
+<InfoBox>
+
+For production environments, we recommend using three nodes. If you initially set up the gateway with one node, you can resize it at a later time. 
+
+</InfoBox>
+
+
+Palette provides an installer in the form of a docker container that is temporarily deployed on your laptop, workstation or jump box. You can run the installer on any system that has a docker daemon installed and connectivity to Palette and the MAAS identity endpoint.
+
+
+# Prerequisites
+
+- Canonical MAAS installed, set up, and available in your environment.
+
+
+- A linux computer with a docker daemon installed and a connection to Palette and the MAAS endpoint. The installer must be invoked on a linux system.
+
+
+- One IP address for a Kubernetes control plane.
+
+
+- One IP address for a single-node gateway or three IP addresses for a three-node gateway.
+
+
+- Sufficient IPs for application workload services, such as Load Balancer services.
+
+<WarningBox>
+
+By default, the MAAS Kubernetes pack uses 192.168.0.0/16. Ensure that the Pod CIDR range for any clusters you deploy after setting up the PCG do not overlap with the network used by the bare metal machines that MAAS manages.
+
+</WarningBox>
+
+- Minimum capacity requirements as follows:
+
+    <br />
+
+    - One node (no HA): 4 vCPU, 8 GiB Memory, 60 GiB Storage.
+   
+    - Three nodes (HA): 6 vCPU, 12 GiB Memory, 90 GiB Storage.<br /><br />
+
+- An active MAAS API key in the following format:
+
+  ``[consumer_key], [key], and [secret] tokens: API key = '[consumer_key]:[key]:[secret]'``
+
+  <br />
+
+- The DNS server that the installer will use must be able to resolve the public internet names of the machines that MAAS manages so it can connect to them.  
+
+    This is commonly done by ensuring the DNS server delegates the MAAS domain to the MAAS control plane. To connect, the installer uses the FQDN ``machine-hostname.maas``. A common way to enable this is to ensure the DNS server delegates the MAAS domain to the MAAS control plane.
+
+# Understand the Gateway Installation Process
+
+The following steps outline the overall process to install the PCG. For detailed steps, refer to [Install the Gateway](/clusters/data-center/maas/install-manage-maas-pcg#installthegateway) below.
+
+<br />
+
+1. You obtain a pairing code in Palette that you will use later.
+
+
+2. Then you run a configuration script using a docker image. 
+
+
+3. The script creates a PCG configuration file.
+
+
+4. Next, you run the installer using the open-source Kind tool and provide the pairing code you obtained in step **1**. The installer uses the configuration file from script output in step **3**.
+
+    The installer needs access to your Palette account and to one or three machines in your MAAS cluster. If you select one machine in Palette, then you need one in MAAS. Likewise, if you select three machines in Palette, you need three in MAAS. The MAAS machines must have internet access and be in a ready state.
```

Comment:
@ritawatson I spoke to Tyler, and he pointed out that there is no need to break out the steps. The install can be done in one step. So we can remove steps 3 and 4.  We just need to update the install command and remove the `-o` flag.

Reply:
Removed steps 3 and 4. Kept last paragraph with step 2 per Tyler. He says this is useful info.